data/fibrousfinance.json

```diff
@@ -32,9 +32,9 @@
   },
   "tokens": [],
   "media": {
-    "logoUrl": "https://i.hizliresim.com/5ne373k.jpg",
-    "bannerUrl": "https://i.hizliresim.com/r5uy2xt.png",
-    "previewUrl": "https://i.hizliresim.com/r5uy2xt.png",
+    "logoUrl": "https://github.com/Fibrous-Finance/fibrous-landing/blob/main/src/assets/ffpp.png",
+    "bannerUrl": "https://github.com/Fibrous-Finance/fibrous-landing/blob/main/src/assets/ffbanner.jpg",
+    "previewUrl": "https://github.com/Fibrous-Finance/fibrous-landing/blob/main/src/assets/ffbanner.jpg",
```

Comment:
These new URLs don'e seem to work properly:
![Screenshot 2022-10-10 at 12 26 58](https://user-images.githubusercontent.com/10310633/194846180-71a7f3b9-49c4-4688-b6db-a37f01ddbb06.png)